pronto-rails_migrations.gemspec

```diff
@@ -20,8 +20,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency('pronto', '~> 0.9.0')
+  spec.add_dependency('pronto', '~> 0.10.0')
```

Comment:
Maybe make this one more permissive as well? `>= 0.10.0`

Reply:
All pronto plugins follow the same pattern to be versioned exactly to the current pronto version :)
 
If you insist I can change it!

Reply:
Ok, lets keep it this way then.